OpenSim/Common/ModelDisplayHints.h

```diff
@@ -53,6 +52,7 @@ your component produces. The currently-supported flags are:
   - show frames
   - show labels
   - show debug geometry
+  - display_scale_factor
```

Comment:
The flags actually have "_" so I'd lean to add those rather than remove the ones in display_scale_factor. I sure will take your wording.